packages/r/rabbitmq-c/xmake.lua

```diff
@@ -0,0 +1,41 @@
+package("rabbitmq-c")
+
+    set_homepage("https://github.com/alanxz/rabbitmq-c")
+    set_description("This is a C-language AMQP client library for use with v2.0+ of the RabbitMQ broker.")
+
+    set_urls("https://github.com/alanxz/rabbitmq-c/archive/refs/tags/$(version).zip",
+             "https://github.com/alanxz/rabbitmq-c.git")
+
+    add_versions("v0.11.0", "fad876075cbcf9a2f3dff27be66391f69117d5068529ba1d53e67fe59c2d88e1")
+
+    add_deps("cmake")
+    add_configs("shared", {description = "Build shared library.", default = false, type = "boolean"})
```

Comment:
why? it only support static library?

Reply:
Sorry! I mistake the use of this config. There is still some problem, I close this pr for the moment.